README.asciidoc

```diff
@@ -1027,13 +1005,13 @@ footnote to a particular line of code:
 ==================================
 
 [[sense-snippets]]
-=== View in Sense
+=== View in Console
```

Comment:
These aren't related but they jumped out at me while I was here. I can break this out if folks are more comfortable with that.

Reply:
Grep tells me there are two more SENSEs lurking in here if you wanna smack 'em.

```
$ grep -iFn sense README.asciidoc
1066:    // SENSE: path/to/snippet.json
1082:// SENSE:one/two/three/example_1.json <1>
```